server/database/queries/captain/editCaptain.js

```diff
@@ -0,0 +1,20 @@
+const connection = require('../../config/dbConnection');
+
+const editCaptain = (captainInfo, id) => {
+  const queryValues = Object.keys(captainInfo).map(info => captainInfo[info]);
```

Comment:
@naremanhilles I think this could give you an unexpected order of keys in the future, and break the query.  It might be better to be explicit:
`const { name, email, mobile_number, .............. } = captainInfo`

Then you could be clearer in your if-statement about which value you are checking:
`if ( !password ) {`